R/constants.R

```diff
@@ -0,0 +1,33 @@
+## ----
+# Specified BrAPI endpoints
+BRAPI_ENDPOINTS <- list(
+    "METHOD_TABLE"   = "allelematrix",
+    "SAMPLES"        = "samples",
+    "SERVER_INFO"    = "serverinfo",
+    "VARIANT_TABLES" = "variantTables",
+    "VARIANTS"       = "variants"
+)
```

Comment:
Does this list imply that all of these endpoints are supported?  What are your allelematrix vs variantTables endpoints?  Are these old endpoints from v1?  We currently only support sample, serverinfo, variants, and soon variantSets (the latter gives you a link to a multi-sample vcf file)

Reply:
This is another copy from v1. I made this because I (1) wanted a simple data structure to point to all possible BrAPI connections since a lot of these are used in other downstream functions. I can probably remove a majority of these at a later date once we finalize v2 endpoints.